<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dbeditdialog</class>
 <widget class="QDialog" name="dbeditdialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>309</width>
    <height>230</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>309</width>
    <height>230</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>309</width>
    <height>230</height>
   </size>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <widget class="QLineEdit" name="e_name">
   <property name="geometry">
    <rect>
     <x>114</x>
     <y>9</y>
     <width>190</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="e_host">
   <property name="geometry">
    <rect>
     <x>114</x>
     <y>63</y>
     <width>190</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="e_db">
   <property name="geometry">
    <rect>
     <x>114</x>
     <y>93</y>
     <width>190</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="e_user">
   <property name="geometry">
    <rect>
     <x>114</x>
     <y>123</y>
     <width>190</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="e_pass">
   <property name="geometry">
    <rect>
     <x>114</x>
     <y>153</y>
     <width>190</width>
     <height>22</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>12</y>
     <width>66</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Название:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>66</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Адрес:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>96</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>База данных:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>126</y>
     <width>96</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Пользователь:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>156</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Пароль:</string>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="b_okcancel">
   <property name="geometry">
    <rect>
     <x>132</x>
     <y>204</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QCheckBox" name="c_savepass">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>177</y>
     <width>146</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Сохранять пароль</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>39</y>
     <width>103</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Идентификатор:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="e_ident">
   <property name="geometry">
    <rect>
     <x>114</x>
     <y>36</y>
     <width>190</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>e_name</tabstop>
  <tabstop>e_ident</tabstop>
  <tabstop>e_host</tabstop>
  <tabstop>e_db</tabstop>
  <tabstop>e_user</tabstop>
  <tabstop>e_pass</tabstop>
  <tabstop>c_savepass</tabstop>
  <tabstop>b_okcancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
 <designerdata>
  <property name="gridDeltaX">
   <number>3</number>
  </property>
  <property name="gridDeltaY">
   <number>3</number>
  </property>
  <property name="gridSnapX">
   <bool>true</bool>
  </property>
  <property name="gridSnapY">
   <bool>true</bool>
  </property>
  <property name="gridVisible">
   <bool>true</bool>
  </property>
 </designerdata>
</ui>
